Python/optimizer.c

```diff
@@ -1161,7 +1164,21 @@ _Py_Executors_InvalidateAll(PyInterpreterState *interp)
         exec->vm_data.valid = false;
         exec->vm_data.linked = false;
         exec = next;
-        OPT_STAT_INC(executors_invalidated);
```

Comment:
We still want stats for this, as it can be called when the builtins are modified, or other rare events.
Maybe add a flag parameter to distinguish between shutdown and actual invalidation?

Reply:
My approach was to move the counters to the callers, rather than a flag (but a flag is fine too if you prefer).  In either case, it looks like I missed `_PyInterpreterState_SetEvalFrameFunc`: This counter should also be incremented there.